pkgs/applications/virtualization/virtualbox/default.nix

```diff
@@ -104,6 +104,7 @@ in stdenv.mkDerivation {
     })
   ++ [
     ./qtx11extras.patch
+    ./linux-5.10-r0drv-memobj-fix-r0.patch
```

Comment:
It would be good to have a comment here to the VBox issue, so people can find the reason for this patch without digging too much when it starts to not apply anymore.

Reply:
Ok!